azure-pipelines.yml

```diff
@@ -96,28 +96,35 @@ stages:
       displayName: 'CR-QC: Static analysis (flake8)'
 
   - job: codecov
-    displayName: Code Coverage
+    displayName: 'Code Coverage'
+    continueOnError: true
     pool:
       vmImage: 'ubuntu-latest'
 
     steps:
+    - pwsh: write-output "project $Env:project, pipeline $Env:pipeline"
+      displayName: 'Helpppppp'
```

Comment:
Are the multiple p's intentional?

Reply:
I'm just shooting in the dark at this point trying to figure out what's going on... hence the work in progress label.